cmd/query/app/server.go

```diff
@@ -219,6 +222,12 @@ func createHTTPServer(
 			MeterProvider:  telset.MeterProvider,
 		},
 		handler,
+		xconfighttp.WithOtelHTTPOptions(
+			otelhttp.WithFilter(func(r *http.Request) bool {
+				ignorePath := path.Join("/", queryOpts.BasePath, "static")
```

Comment:
Can `queryOpts.BasePath` be `/` here? If so it will create invalid `//static` prefix.

Reply:
@yurishkuro yeah it can - `path.Join` removes duplicate slashes. The leading slash needs to be there in the case where `queryOpts.BasePath` is empty because a slash doesn't get added in that case.